docs/MAAP_USAGE.md

```diff
@@ -145,14 +132,41 @@ To run a GEDI subsetting DPS job, you must supply the following inputs.
   Examples showing how the value specified for `output` is mapped to a final
   output file:
 
-  - Unspecified -> `myaoi.gpkg`, where `myaoi.geojson` is the name of the AOI file
+  - Unspecified -> `myaoi.gpkg`, where `myaoi.geojson` is the name of the AOI
+    file
   - `myoutput` -> `myoutput.gpkg`
   - `myoutput.gpkg` -> `myoutput.gpkg`
   - `myoutput.h5` -> `myoutput.gpkg`
   - `mypath/myoutput` -> `mypath/myoutput.gpkg`
 
   _New in version 0.6.0_
 
+- `scalene_args` (_optional_): Arguments to pass to [Scalene] for performance
```

Comment:
Suggested rewrite for this section to emphasize what Science users need to know first, developer and more advanced information later.

```
- `scalene_args` (_optional_): Arguments to pass to [Scalene] for performance
  profiling. Normal usage should leave this argument blank.
  Fill this in if you want to collect performance metrics (i.e. CPU and RAM usage)
  The recommended value for this input is  `--reduced-profile` (see below for more advanced usage).
  When used you will find `profile.html` in your algorithm output folder.

  When this input is supplied, the algorithm will be run via the `scalene` command, and the value of this input will be passed as arguments to
  the command.  For a list of the available command-line options, see
  <https://github.com/plasma-umass/scalene?tab=readme-ov-file#scalene>.

  Starting with `--reduced-profile` produces a relatively brief report that may aid in more quickly identifying hotspots than a fully profile would.  However, to produce a full profile where you want to use all of Scalene's default values, you must supply _some_ value for this input, so the simplest valid Scalene option is `--on`.  Otherwise, as mentioned above, when no value is supplied for this input, Scalene will not be used at all.

  > **Note:** that since no browser is available in DPS, when any value is supplied for this input, the `--no-browser` option will be included to prevent Scalene from attempting to open a browser.  However, the `--web`
  option will also be included, which will produce HTML output to a file named `profile.html`.
```

Reply:
Will do.